.gitignore

```diff
@@ -1,2 +1,3 @@
 /target/
 **/*.rs.bk
+/.vscode
```

Comment:
I'd recommend adding this to your global .gitignore instead of adding it to the specific project's .gitignore https://gist.github.com/subfuzion/db7f57fff2fb6998a16c